components/script/Cargo.toml

```diff
@@ -129,6 +129,11 @@ webrender_api = { workspace = true }
 webrender_traits = { workspace = true }
 webxr-api = { workspace = true, features = ["ipc"], optional = true }
 xml5ever = { workspace = true }
+nom = "7.1.3"
 
 [target.'cfg(not(target_os = "ios"))'.dependencies]
 mozangle = { workspace = true }
+
+[dev-dependencies]
+insta = "1.41.1"
+rstest = "0.23.0"
```

Comment:
I made some snapshot tests for the parser, although I know opinions are divided on whether they are useful or not. Happy to take them out.

Reply:
I think we should avoid adding more dependencies just for some tests. I'm neutral on `nom` for the moment as it seems it is already a dependency, but I would really appreciate it if this change added a minimal number of new ones.

Reply:
Fair enough, I'll simplify the tests!